pkg/templates/notificationtemplates/notification_generator.go

```diff
@@ -1,18 +1,26 @@
 package notificationtemplates
 
 import (
+	"embed"
+	"fmt"
+	"io/fs"
 	"strings"
 
-	"github.com/codeready-toolchain/host-operator/pkg/templates/assets"
-
+	"github.com/codeready-toolchain/host-operator/deploy"
 	"github.com/pkg/errors"
 )
 
+const (
+	SandboxNotificationTemplateSetName   = "sandbox"
+	AppstudioNotificationTemplateSetName = "appstudio"
+	UserProvisionedTemplateName          = "userprovisioned"
+	UserDeactivatedTemplateName          = "userdeactivated"
+	UserDeactivatingTemplateName         = "userdeactivating"
+	IdlerTriggeredTemplateName           = "idlertriggered"
```

Comment:
would it make sense to move it either to api repo or toolchain-common repo next to the `NotificationBuilder` so we use the constant also in member operator? https://github.com/codeready-toolchain/member-operator/blob/master/controllers/idler/idler_controller.go#L208
Can be done in separate PR :wink:

Reply:
yeah, it makes absolute sense